recipes/opencv/3.x/conanfile.py

```diff
@@ -0,0 +1,395 @@
+        self._cmake.definitions["WITH_OPENCL_SVM"] = False
+        self._cmake.definitions["WITH_OPENGL"] = False
+        self._cmake.definitions["WITH_OPENNI"] = False
+        self._cmake.definitions["WITH_OPENNI2"] = False
+        self._cmake.definitions["WITH_OPENVX"] = False
+        self._cmake.definitions["WITH_PLAIDML"] = False
+        self._cmake.definitions["WITH_PROTOBUF"] = False
+        self._cmake.definitions["WITH_PTHREADS_PF"] = False
+        self._cmake.definitions["WITH_PVAPI"] = False
+        self._cmake.definitions["WITH_QT"] = False
+        self._cmake.definitions["WITH_QUIRC"] = False
+        self._cmake.definitions["WITH_V4L"] = False
+        self._cmake.definitions["WITH_VA"] = False
+        self._cmake.definitions["WITH_VA_INTEL"] = False
+        self._cmake.definitions["WITH_VTK"] = False
+        self._cmake.definitions["WITH_VULKAN"] = False
+        self._cmake.definitions["WITH_XIMEA"] = False
+        self._cmake.definitions["WITH_XINE"] = False
+        self._cmake.definitions["WITH_LAPACK"] = False
+        self._cmake.definitions["WITH_IPP_IW"] = False
+        self._cmake.definitions["WITH_CAROTENE"] = False
+        self._cmake.definitions["WITH_PROTOBUF"] = False
+        self._cmake.definitions["WITH_LAPACK"] = False
+
+        self._cmake.definitions["WITH_JPEG"] = self.options.with_jpeg
+        self._cmake.definitions["WITH_PNG"] = self.options.with_png
+        self._cmake.definitions["WITH_TIFF"] = self.options.with_tiff
+        self._cmake.definitions["WITH_JASPER"] = self.options.with_jasper
+        self._cmake.definitions["WITH_OPENEXR"] = self.options.with_openexr
+        self._cmake.definitions["WITH_EIGEN"] = self.options.with_eigen
+        self._cmake.definitions["WITH_WEBP"] = self.options.with_webp
+        self._cmake.definitions["WITH_DSHOW"] = self.settings.compiler == "Visual Studio"
+        self._cmake.definitions["WITH_MSMF"] = self.settings.compiler == "Visual Studio"
+        self._cmake.definitions["WITH_MSMF_DXVA"] = self.settings.compiler == "Visual Studio"
+        self._cmake.definitions["WITH_GTK"] = self.options.get_safe("with_gtk", False)
+        self._cmake.definitions["WITH_GTK_2_X"] = self.options.get_safe("with_gtk", False)
+
+        self._cmake.definitions["OPENCV_MODULES_PUBLIC"] = "opencv"
+        if self.options.parallel:
+            self._cmake.definitions["WITH_TBB"] = self.options.parallel == "tbb"
+            self._cmake.definitions["WITH_OPENMP"] = self.options.parallel == "openmp"
+
+        if self.options.contrib:
+            self._cmake.definitions["OPENCV_EXTRA_MODULES_PATH"] = os.path.join(self.build_folder, self._contrib_folder, 'modules')
+
+        if self.settings.compiler == "Visual Studio":
+            self._cmake.definitions["BUILD_WITH_STATIC_CRT"] = self.settings.compiler.runtime in ("MT", "MTd")
+        if self.options.with_openexr:
+            self._cmake.definitions["OPENEXR_ROOT"] = self.deps_cpp_info['openexr'].rootpath
```

Comment:
This is needed on Windows with backslashes in the path.
```suggestion
            self._cmake.definitions["OPENEXR_ROOT"] = self.deps_cpp_info['openexr'].rootpath.replace("\\", "/")
```